spec/controllers/hyrax/admin/admin_sets_controller_spec.rb

```diff
@@ -174,31 +174,32 @@
       end
 
       describe "#create" do
-        before do
-          controller.admin_set_create_service = service
-        end
-
         context "when it's successful" do
-          let(:service) do
-            lambda do |admin_set:, **_kargs|
```

Comment:
The controller now calls the `.call!` method instead of the deprecated `.call` method.  The lambda service expects to receive `call`, and as such won't work anymore.

Reply:
what motivated the switch to `#call!`?

this seems like a downside of that move. i'm not necessarily wanting to revisit if it's already done, but am curious.

Reply:
The create service saves the admin set.  

For ActiveFedora:  `admin_set.save` updates `admin_set`.  

For Valkyrie:  `Hyrax.persister.save(resource: admin_set` does not, so you have to return the results of the save to have access to the updated.

For backward compatibility, `.call` continues to return Boolean.  The `.call` method is deprecated. `.call!` is the replacement method and it returns the updated admin set.

Reply:
thanks! this refreshed my memory.